lib/Cargo.toml

```diff
@@ -60,10 +61,14 @@ winapi = { version = "0.3", features = [
 ] }
 
 # Unix
-[target.'cfg(not(windows))'.dependencies]
-nix = "0.24"
+[target.'cfg(unix)'.dependencies]
 whoami = "1"
 
-# Linux / Freebsd
-[target.'cfg(any(target_os = "linux", target_os = "freebsd"))'.dependencies]
-procfs = { version = "0.14", default-features = false }
+# Linux only
+[target.'cfg(target_os = "linux")'.dependencies]
+procfs = "0.14.1"
+
+# Apple only
+[target.'cfg(target_vendor = "apple")'.dependencies]
+# fork that fixes a version conflict, see heim-rs/darwin-libproc#3
+darwin-libproc = { version="0.2.0", git = "https://github.com/jamwaffles/darwin-libproc.git", rev = "cbd571e4be87a6603de328d1f1f0ad54d7bdfedf" }
```

Comment:
This might actually be a blocker for publishing a new release.
One isn't allowed to publish packages to crates.io that depend on git repositories.

As far as I know, the version `0.2.0` will then be used as a fallback, effectlively breaking this release for all apple platforms.

Reply:
The darwin-libproc library also seems quite abandoned.

Do you think there's any chance on this getting fixed in time?

Reply:
🤦 I'll see if https://github.com/andrewdavidmackenzie/libproc-rs can replace it. I was under the impression that heim was an active project, and darwin-libproc is part of heim. Turns out I was wrong about that :-(

Reply:
libproc-rc is missing support for enumerating processes in a process group, see andrewdavidmackenzie/libproc-rs/issues/85 for my request that that be added.

Reply:
Actually, this is a **test** dependency only now. I'll rework it as such.

>  One isn't allowed to publish packages to crates.io that depend on git repositories.

Can you publish a package where only the test dependency is a git repo?

Reply:
I found confirmation that git dev-dependencies are ignored, so I can make this work. Force push incoming.